docs/index.md

```diff
@@ -1543,3 +1587,31 @@ Pass it 3 of your Setoid-implementing instances and the Setoid laws will be chec
 ##### Works with functions
 
 - [equals](#equals)
+
+#### Ord
+
+Encapsulates order. Given two objects of type `Ord`, you can determine which one is the greater of the two.  
+Extends [Setoid](#setoid).
+
+##### Methods
+
+```
+public function lte(Ord $that): bool;
+```
+
+##### Testing
+
+```
+use Garp\Functional\Types\Traits\TestsOrdLaws;
+
+$this->assertObeysOrdLaws($ord1, $ord2, $ord3);
+```
+
+Pass it 3 of your Ord-implementing instances and the Ord laws will be checked against your class.
+
+##### Works with functions
+
+- [lt](#lt)
+- [lte](#lte)
+- [gt](#gt)
+- [gte](#gte)
```

Comment:
`Ord` extends `Setoid` so `equals` is could be added to this function aswel.

Reply:
Check, fair point. 👍